content/tutorials/manage-the-rippled-server/install-rippled.md

```diff
@@ -46,59 +81,193 @@ This section assumes that you are using CentOS 7 or Red Hat Enterprise Linux 7.
 
         $ sudo systemctl start rippled.service
 
-## Installation on Ubuntu with alien
+For next steps, see [Postinstall](#postinstall) and [Additional Configuration](#additional-configuration).
 
-This section assumes that you are using Ubuntu 15.04 or later.
 
-1. Install yum-utils and alien:
+## Installation on macOS
 
-        $ sudo apt-get update
-        $ sudo apt-get install yum-utils alien
+At this time, we don't recommend using the macOS platform for `rippled` production use. For production, consider using the [Ubuntu platform](#installation-on-ubuntu-with-alien), which has received the highest level of quality assurance and testing.
 
-2. Install the Ripple RPM repository:
+That said, macOS is suitable for many development and testing tasks.
 
-        $ sudo rpm -Uvh https://mirrors.ripple.com/ripple-repo-el7.rpm
+This section assumes that you are using macOS X 10.0 or higher. ***TODO: okay?***
 
-3. Download the `rippled` software package:
+{% set n = cycler(* range(1,99)) %}
 
-        $ yumdownloader --enablerepo=ripple-stable --releasever=el7 rippled
+{{n.next()}}. Install [Xcode](https://developer.apple.com/download/).
 
-4. Verify the signature on the `rippled` software package:
+{{n.next()}}. Install Xcode command line tools. ***TODO: Comment from PM: Installing homebrew removes the need to install xcode command line tools separately. JHA: homebrew lists xcode command line tools as a requirement: https://docs.brew.sh/Installation#requirements. Do we keep or remove this step?***
 
-        $ sudo rpm --import https://mirrors.ripple.com/rpm/RPM-GPG-KEY-ripple-release && rpm -K rippled*.rpm
+        xcode-select --install
 
-5. Install the `rippled` software package:
+{{n.next()}}. Install [Homebrew](https://brew.sh/).
 
-        $ sudo alien -i --scripts rippled*.rpm && rm rippled*.rpm
+        ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 
-6. Configure the `rippled` service to start on system boot:
+{{n.next()}}. Update Homebrew.
 
-        $ sudo systemctl enable rippled.service
+        brew update
 
-7. Start the `rippled` service
+{{n.next()}}. Use Homebrew to install dependencies. ***TODO: Added boost per PM's request: Installing Boost directly is going to frustrate people - brew can do it in one command. I ended up installing Boost with Brew with no issues. Which release to choose? JHA: I did `brew install boost` and brew installs 1.67.0 in /usr/local/Cellar/boost/1.67.0_1 -- is this okay? If we install Boost via brew, do we no longer have to compile boost: ./bootstrap.sh and ./b2 cxxflags="-std=c++14"? Comment from PM: Build issues with Boost not finding lib (boost-context) in my case. JHA: when does this issue come up? How can we help the user avoid or address this issue?***
```

Comment:
(1) Using homebrew removes the need to compile it from scratch. You also do not need to set environment variables yourself. The only catch is brew updates to the latest version of boost, which the Rippled team must first OK in order for it to work. 1.68 is the latest release and it works fine, but 1.69 will need to be checked. We may need to periodically update to reflect this?

(2) Not sure why it didn't work, I did not try to debug it and this is why I tried to use homebrew instead.

Reply:
Users will have to compile boost themselves any time the version of boost that's in the homebrew repositories is not a version that's supported by rippled. At the moment the two are compatible but I'm not convinced that's going to remain true for the long term. Perhaps we should have both instructions available and switch the recommendations periodically based on whether the easier brew install is compatible.

Reply:
And as Daniel alluded, the compatibility between the Brew and rippled versions can break in two ways:

- brew updates Boost to a new version that's not compatible with the current release of rippled
- rippled updates to a new version that's not compatible with the version of Boost in the Brew repositories

The former is more likely than the latter since Brew tends to update quickly, but the latter is also possible if the person who's currently maintaining the Boost package for Brew changes or takes a long vacation or something, or if the new version of Boost is held up from being added to Brew because of some other incompatibilities with other Brew software that requires it. (Similar situations have happened to me on Arch Linux.)

The "compile it yourself" instructions should be fairly stable and will only change as the result of `rippled` updates, so that would be a much easier maintenance path going forward, even though it's more work for users. The "use the version from Brew" instructions are only sometimes going to be possible, but when they work, they are much better for users.

Reply:
Its a tricky thing to maintain the brew command that could go out of date. Im not sure how often Boost updates, but thats about how often we would need to update these instructions with the new version. You can have brew install a specific version of a formula, which is the 3rd option.